src/bridge/ExposureNotification/types.ts

```diff
@@ -75,7 +76,7 @@ export interface ExposureConfiguration {
 }
 
 export interface DailySummariesConfig {
-  reportTypeWeights: Record<Report, number>;
+  reportTypeWeights: Record<ReportType, number>;
```

Comment:
I think the `number` type can be removed for this and the next line.

Reply:
The type definition for `DailySummariesConfig` is entirely unused and I don't think we'll need it. So I'll remove the whole thing.